### PR TITLE
Profile fix

### PR DESCRIPTION
**Pull request by bricehalder**
Fills profile page with relevant user data

Screen Shot 2022-02-24 at 8 38 11 PM

### CODE REVIEW

File: apis/profiles.py [hunk 1]
```diff
@@ -13,11 +13,20 @@ def get_user_info(username):
...
-    if (session.get("username") == username or session.get("admin")):
-        query = "SELECT `username`, `email`, `email_confirmed`, `admin` FROM `users` WHERE `username`=%s"
-    else:
-        query = "SELECT `username`, `email_confirmed`, `admin` FROM `users` WHERE `username`=%s"
 
+    admin_query_string = 'admin,' if session.get("username") == username or session.get("admin") else ''
+    query = f'''
+    SELECT
+        username,
+        email,
```
Comment (dqian3) [inline apis/profiles.py:21]: We shouldn't expose people's emails

Comment (dqian3) [reply apis/profiles.py:21]: that was the point of only querying for email if the user it itself.

**Review by dqian3 — CHANGES_REQUESTED** [review]

File: apis/profiles.py (hunk 1 above)
Comment (bricehalder) [reply apis/profiles.py:21]: Oh oops, yeah this was unintentional good catch

Comment (bricehalder) [reply apis/profiles.py]: fixed

**Review by dqian3 — APPROVED** [review]

**Pull request merged**